helloworld/greet/BUILD

```diff
@@ -3,13 +3,19 @@
 
 # This target sets the metadata for all the Python non-test files in this directory.
 #
-# * `name` defaults to the name of this directory, i.e., `greet`.
-# * `sources` defaults to ['*.py', '!*_test.py', '!test_*.py', '!conftest.py'].
-# * `dependencies` are inferred.
-python_library()
+# * Pants cannot infer dependencies on `resources` targets, so we explicitly add the dep.
+python_library(
+    name="lib",
+    dependencies=[":translations"],
+)
 
 # This target sets the metadata for all the Python test files in this directory.
-#
-# * `sources` defaults to ['*_test.py', 'test_*.py', 'conftest.py'].
-# * `dependencies` are inferred.
-python_tests(name='tests')
+python_tests(
+    name="tests",
+)
```

Comment:
I very much prefer the style of keeping it on one-line. But `tailor` generates it this way, so probably good to be consistent with that.